quickstart.md

```diff
@@ -115,12 +115,27 @@ Details on each of the runtime classes:
 - *kata-qemu-tdx* - using QEMU, with TDVF, and support for Intel TDX CC HW
 - *kata-qemu-sev* - using QEMU, and support for AMD SEV HW
 
+For the process based CoCo TEE (aka. `enclave-cc`) the operator setup steps are the same but instead
+of `ccruntime.yaml`, either `ccruntime-enclave-cc-sim.yaml` or `ccruntime-enclave-cc.yaml` for the
+*simulated* SGX mode build or *hardware* SGX mode build, respectively, should be used.
```

Comment:
```suggestion
**simulated** SGX mode build or **hardware** SGX mode build, respectively, should be used.
```

I'd make simulataed and hardware bold, rather than italic.